src/test/java/com/karankumar/booksapi/repository/BookRepositoryTest.java

```diff
@@ -105,8 +107,47 @@ private Book createBookwithISBN() {
         book.setIsbn13(ISBN);
         return book;
     }
```

Comment:
As per our style guide, could you please have one blank line between the end of one method and the start of the next?

Also, could you please ensure `@Test` is correctly indented (in line with `@DisplayName`)?